Octokit/Models/Request/SearchRepositoriesRequest.cs

```diff
@@ -304,33 +304,43 @@ public class DateRange
         /// Matches repositories with regards to the <param name="date"/>.
         /// We will use the <param name="op"/> to see what operator will be applied to the date qualifier
         /// </summary>
-        public DateRange(DateTime date, SearchQualifierOperator op)
+        [Obsolete("This ctor has been deprecated as GitHub API now supports date and time")]
+        public DateRange(DateTime date, SearchQualifierOperator op) : this(new DateTimeOffset(date), op)
```

Comment:
By forwarding these ctors to the new `DateTimeOffset` overloads, the behavior has changed by taking the time into account. That could be a breaking change.

I think the safest thing to do is to keep the existing logic (as @ryangribble noted in #1904) for these ctors and use the new (and improved) format for the new `DateTimeOffset` ctors.